src/fabio_rodrigues_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,55 @@
+from random import randint
+
+__author__ = 'FABIO RODRIGUES PEREIRA'
+__email__ = 'faro@nmbu.no'
+
+
+def game(max_attempts):
+    """It is a game where you have x-max_attempts to guess right the sum of 2 dice.
+    Your score will be the x-max_attempts minus the number of mistakes you had (attempted).
+    You lose the game when reach 0 score.
+    """
+    num_to_guess = sum_dice()
+
+    for attempted in range(max_attempts):
+        right_guess = guess_check(guessed_num(), num_to_guess)
+
+        if not right_guess and attempted < max_attempts-1:
+            print('Wrong, try again!')
+
+        elif right_guess and attempted < max_attempts:
+            print('You won {} points.'.format(score(max_attempts, attempted)))
+            break
+
+        else:
+            print('You lost. Correct answer: {}.'.format(num_to_guess))
+            break
+
+
+def sum_dice():
+    """Returns the sum of 2 rolled random dice
+    """
+    return randint(1, 12)
```

Comment:
The sum of 2 rolled random dice cannot be 1 😄 
Also, the probability is wrong here.
For `randint(2, 12)`, the probability to get 2 or 12 is 1/11, while `randint(1, 6) + randint(1, 6)` the probability to get 2 or 12 is 1/36 😂